frontend/ui/src/app/components/users/users.component.ts

```diff
@@ -153,17 +159,15 @@ export class UsersComponent implements OnDestroy {
 
   public async deleteUser(user: UserAccountWithRole): Promise<void> {
     this.overlay
-      .confirm(`Really delete ${user.name ?? user.email}?`)
+      .confirm(`This will remove ${user.name ?? user.email} from your organization. Are you sure?`)
       .pipe(
         filter((result) => result === true),
         switchMap(() => this.users.delete(user)),
         catchError((e) => {
-          if (e instanceof HttpErrorResponse && e.status === 400) {
-            this.toast.error(
-              `User ${user.name ?? user.email} cannot be deleted.
-              Please ensure there are no deployments managed by this user and try again.`
-            );
-          }
+          const msg = getFormDisplayedError(e);
+          if (msg) {
+            this.toast.error(msg);
+          } // ??
```

Comment:
please remove or explain :-)
```suggestion
          }
```